pvt/benchmarks/src/PoolBenchmark.behavior.ts

```diff
@@ -443,6 +493,10 @@ export class Benchmark {
       });
 
       it(`measures gas (unbalanced) (${poolTag})`, async () => {
+        if (this._settings.disableUnbalancedLiquidityTests) {
+          return;
```

Comment:
here and elsewhere: I think you can do `this.skip()` inside `it` in mocha: https://stackoverflow.com/a/32820119

That will show the test as skipped when you run it instead as successful, which reflects what we're doing a bit better.